src/Test/StagingStorageLoader.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbWriter\Synapse\Test;
+
+use Keboola\DbWriter\Synapse\Application;
+use RuntimeException;
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\Options\GetFileOptions;
+
+class StagingStorageLoader
+{
+    private const CACHE_PATH = __DIR__ . '/../../tests/.cache/StagingStorageLoader.cache.json';
```

Comment:
Kedze je to pomale, tak sa to cachuje,.... pripade nejakej zmeny v datach si treba cache subor manualne vymazat.